content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst

```diff
@@ -2,35 +2,46 @@
 Invoice Time Spent on Tickets (Postpaid Support Services)
 =========================================================
 
-Directly pull the billable time you have tracked on your helpdesk tickets into sales orders and
-invoices through a project task. It gives you more control over what you charge your client, and it
-is more efficient.
+Connect your Helpdesk ticket to a Project task and use either one to record timesheets that you can later pull from to bill your client. This gives you more control over what you charge your client, and it is more efficient.
 
 Configuration
 =============
+Go to :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams` and select the Helpdesk team you want to track timesheets for. Click **Edit** and enable the options Timesheet on Ticket and Time Reinvoicing.
 
-| Go to :menuselection:`Helpdesk --> Configuration --> Helpdesk team --> Edit` and enable the options
-  *Timesheet on Ticket* and *Time Reinvoicing*.
-| Under *Timesheet on Ticket*, choose the *Project* to which tickets (and timesheets) will be
-  linked by default. Open its *External link* to enable the feature *Bill from tasks*.
+Under Timesheet on Ticket, choose the Project to which tickets (and timesheets) will be linked by default. Open the Project's external link to edit its settings and enable the Billable option. Save both the Project and the Helpdesk team's settings to apply these features.
```

Comment:
Contextualize this. "Under Timesheet on Ticket..." could mean many things. Under _what_ exactly?

```suggestion
In the Timesheet on Ticket options
```